### ISSUE TITLE
TodoCommentChecker false positives

### ISSUE THREAD

**Issue by dragisak** [scalastyle/scalastyle#181]
Using sbt-scalastyle 0.8.0

In scalastyle-config.xml:

``` xml
<check level="warn" class="org.scalastyle.scalariform.TodoCommentChecker" enabled="true">
  <parameters>
   <parameter default="TODO|FIXME" type="string" name="words"/>
  </parameters>
</check>
```

I get warning for lines with normal comment. For example, this source will return a warning: `[warn] /Users/dragisak/Ticketfly/RPT/app/com/ticketfly/rpt/validators/StringValidations.scala:16:45: Use of TODO/FIXME comment`

``` scala
package com.ticketfly.rpt.validators

import com.ticketfly.rpt.model._
import org.joda.time.base.AbstractPartial
import org.joda.time.format.DateTimeFormat

import scalaz.Scalaz._

object StringValidations {

  private val SEPERATOR = ','

  def parseEventIdsComma(s: String): Check[Array[EventID]] = try {
    val arr = for {
      str <- s.split(SEPERATOR)
      trimmed = str.trim if trimmed.nonEmpty // ignore white spaces
      eventId = trimmed.toLong
    } yield eventId

    arr.successNel
  } catch {
    case badFormat: IllegalArgumentException => TextMessage("eventid.should.be.number", s).failureNel
  }
}
```

**Comment by alexeykiselev** [comment]
+1
If you put "TODO:" in your comment you won't get a warning.

**Comment by baatout** [comment]
alexeykiselev's solution: (for copy/past purpose)
```
 <check enabled="true" class="org.scalastyle.scalariform.TodoCommentChecker" level="warning">
  <parameters>
   <parameter name="words">TODO|FIXME</parameter>
  </parameters>
 </check>
```

**Comment by matthewfarwell** [comment]
I think the original problem was fixed, so closing this. If there is still a problem, please open another.
